docs/operator_guide/get_started/install_from_source.md

```diff
@@ -94,6 +100,303 @@ meson compile -C build
 
 This is already it!
 
+## Meson Configuration Options
+
+This section describes the configuration options specific to Constellation, their purposes, and how to use them.
```

Comment:
Please remove all options except for

- `buildtype`
- `build_gui`
- `cxx_tests` (mention Catch2)
- `cxx_tools`

Satellites specific options are mentioned in their respective readme, and the options for the subprojects are not of interest.